tests/types/array.test.js

```diff
@@ -15,6 +15,7 @@ describe("ArrayType", function() {
 
     describe("push", () => {
       let pushed;
+
```

Comment:
There are a lot of these extra spaces added, but as far as I can tell, they aren't related to the change. Part of good commit hygiene is to only include those things which pertain to what you're actually trying to accomplish. We can add this space and other in another PR (although should probably be accompanied by getting eslint set up)

Reply:
👍 I've pulled these out for a separate ESLint PR.